tests/api/options.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Options REST API Test
+ *
+ * @package WooCommerce Admin\Tests\API
+ */
+
+use \Automattic\WooCommerce\Admin\API\Options;
+
+/**
+ * WC Tests API Options
+ */
+class WC_Tests_API_Options extends WC_REST_Unit_Test_Case {
+
+	/**
+	 * Endpoints.
+	 *
+	 * @var string
+	 */
+	protected $endpoint = '/wc-admin/v1/options';
+
+	/**
+	 * Setup test data. Called before every test.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->user = $this->factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+	}
+
+	/**
+	 * Test getting options without valid permissions.
+	 *
+	 * @since 3.5.0
```

Comment:
The `@since` here can be dropped.

Reply:
Updated in https://github.com/woocommerce/woocommerce-admin/pull/3004/commits/ba72b7eb457ba83a6b14966c768ba3d2ea8b8c99